packages/design-system/src/styles/styles.scss

```diff
@@ -20,3 +20,5 @@
 @import "theme/oc-visibility";
 @import "theme/oc-width";
 @import "theme/tooltip";
+
```

Comment:
Comment about where the marker is used would be helpful (reference to rollup.config.mjs).

Reply:
Added reference